platform/api/src/database/two_fa_request.rs

```diff
@@ -0,0 +1,8 @@
+use common::database::{Protobuf, Ulid};
+
+#[derive(Debug, Clone, Default, sqlx::FromRow)]
+pub struct TwoFaRequest {
+    id: Ulid,
+    user_id: Ulid,
+    action: Protobuf<()>,
```

Comment:
Why is this a protobuf of unit?

Reply:
It's temporary. I added it so I can check if the rest compiles.